Apps/W1/Email - SMTP Connector/app/src/SMTPMessage.Codeunit.al

```diff
@@ -95,7 +120,7 @@ codeunit 4514 "SMTP Message"
         Recipients: List of [Text];
     begin
         // Add From
-        AddFrom(SMTPAccount."Sender Name", SMTPAccount."Email Address");
+        AddFrom(SMTPAccount);
```

Comment:
There are some changes in #15762 that overlap with this PR. But you could move these to `SMTPConnectorImpl` codeunit in the `BuildSMTPMessage` function first, that'd be great. As this codeunit no longer exists.